### PR TITLE
applications: nrf5340_audio: Add DFU to adapting for end prod doc

### PR DESCRIPTION
**Pull request by erikrobstad**
Change MCUBoot key and create own DFU app

Signed-off-by: Erik Robstad <erik.robstad@nordicsemi.no>

### CODE REVIEW

File: applications/nrf5340_audio/README.rst [hunk 1]
```diff
@@ -1344,6 +1344,13 @@ To use the nRF5340 Audio application with your custom board:
 #. Build the application by selecting the name of the new board (for example, ``new_audio_board_name``) in your build system.
    For example, when building from the command line, add ``-b new_audio_board_name`` to your build command.
 
+FOTA for end products
+=====================
+
+The default MCUBoot key should not be used for end products. See :ref:`west-sign` for more information.
```
Comment (FrancescoSer) [inline applications/nrf5340_audio/README.rst:1350]:
```suggestion
Do not use the default MCUBoot key for end products.
See :ref:`west-sign` for more information.
```

Also isn't a link to ug_fw_update more appropriate? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_fw_update.html

**Review by FrancescoSer — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/README.rst (hunk 1 above)
Comment (erikrobstad) [reply applications/nrf5340_audio/README.rst]: Done. Changed to this link :)

Comment (erikrobstad) [reply applications/nrf5340_audio/README.rst]: Edit: I added both links, since it looks like they contain some different information

**Review by alexsven — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

File: applications/nrf5340_audio/README.rst [hunk 2]
```diff
@@ -1344,6 +1344,13 @@ To use the nRF5340 Audio application with your custom board:
 #. Build the application by selecting the name of the new board (for example, ``new_audio_board_name``) in your build system.
    For example, when building from the command line, add ``-b new_audio_board_name`` to your build command.
 
+FOTA for end products
+=====================
+
+Do not use the default MCUBoot key for end products. See :ref:`ug_fw_update` and :ref:`west-sign` for more information.
```
Comment (FrancescoSer) [inline applications/nrf5340_audio/README.rst:1350]:
```suggestion
Do not use the default MCUBoot key for end products.
See :ref:`ug_fw_update` and :ref:`west-sign` for more information.
```

**Review by FrancescoSer — CHANGES_REQUESTED** [review]

**Review by FrancescoSer — APPROVED** [review]

**Pull request merged**